client/src/app/services/order.service.ts

```diff
@@ -48,32 +48,31 @@ export class OrderService {
     this.cartSubject.next([]);
   }
 
+  //* Get full cart
   getAllItems() {
     return this.httpClient.get<OrderResponse>(this.baseUrlOrder);
   }
 
+  getOrder(id) {
+    return this.httpClient.get(`${this.baseUrlOrder}/invoice/${id}`);
+  }
+
   getTotalCarts() {
-    return this.httpClient.get(`${this.baseUrl}/sum`);
+    return this.httpClient.get(`${this.endpoint}/sum`);
   }
 
   completeOrder(body) {
```

Comment:
return type

Reply:
added generic

Reply:
i dont see it... did you push the commit?

Reply:
yes, maybe it doesnt show it on here